src/bot/commands/user_commands.py

```diff
@@ -76,9 +85,26 @@ def run_get_player_recents_command(update, context):
     if limit > 20:
         limit = 20
 
-    response, status_code = endpoints.get_player_recent_matches_by_account_id(
-        account_id
-    )
+    try:
+        hero_name = " ".join(hero_name_parts)
+        hero = hero_services.get_hero_by_name(hero_name)
+        hero_id = str(hero.id)
```

Comment:
This is redundant, when using the `%s` syntax, it will call the `__repr__` method associated with the type and be converted to a string automatically. This just seems like a weird way to check for a None type to raise an error than explicitly checking for None. Lets clean this up and try to flatten it out.